src/OpenTelemetry.Instrumentation.AspNetCore/README.md

```diff
@@ -116,6 +116,19 @@ Http Semantic Conventions using `OTEL_SEMCONV_STABILITY_OPT_IN`.
 * If user sets the environment variable to `http/dup`, the instrumentation
   emits both `http.server.duration` and `http.server.request.duration`.
 
+##### Metrics on ASP.NET Core 8.0 and above
+
+This library enables all [built-in
```

Comment:
`OTEL_SEMCONV_STABILITY_OPT_IN` -- is this respected for .net8?

Reply:
for tracing it is. not for metrics. Planning to remove for tracing too before next release.

Reply:
i meant to say - its not clear from docs, what is expected behavior in .NET8+newer for metrics.

We could write that .NET8+, AddInst() is simply calling AddMeter(meternames), and nothing more. And explicitly state that no other feature of this package, like the semconv_opt-In is respected in .NET8+.

Reply:
Once we remove semconv_opt-In (which we will [soon](https://github.com/open-telemetry/opentelemetry-dotnet/pull/5066)), there is no additional feature that instrumentation library will provide which will be available in net6.0/net7.0 but not in net8.0. 

> AddInst() is simply calling AddMeter(meternames)

This is an implementation detail, not sure if we need to mention this here.

Reply:
I plan to rewrite this section(in follow up) as well: https://github.com/open-telemetry/opentelemetry-dotnet/blob/8ee44877077d9c44154513a1fd4e1d06fd0b43d5/src/OpenTelemetry.Instrumentation.AspNetCore/README.md#list-of-metrics-produced. That will make it clear what is the difference between lower versions and net8.0.